server/lib/nl/fulfillment/simple.py

```diff
@@ -88,12 +92,15 @@ def populate(state: PopulateState, chart_vars: ChartVars, places: List[Place],
       if chart_type == ChartType.TIMELINE_WITH_HIGHLIGHT:
         sv_place_facet_ids = ext.get_sv_place_facet_ids(chart_vars.svs, places,
                                                         state.exist_checks)
+      sv_place_latest_date = ext.get_sv_place_latest_date(
+          chart_vars.svs, places, None, state.exist_checks)
```

Comment:
hmm, why should we call this always?

Reply:
we only set latest_valid_date in exist_checks if there is a date_range, so this function will map all sv and places to empty string if there was no date range. Added comments in `get_sv_place_latest_date` and `ExistInfo`